internal/service/alarms/api/openapi.yaml

```diff
@@ -34,13 +34,9 @@ paths:
       tags:
         - alarms
       parameters:
-        - in: query
-          name: filter
-          schema:
-            type: string
-          allowReserved: true
-          description: Optional filter parameter. Only the attributes in AlarmEventRecord are filterable.
-          example: (eq,perceivedSeverity,0)
+      - $ref: "../../common/api/openapi.yaml#/components/parameters/excludeFields"
+      - $ref: "../../common/api/openapi.yaml#/components/parameters/fields"
+      - $ref: "../../common/api/openapi.yaml#/components/parameters/filter"
```

Comment:
Ah we totally missed the other params from spec...I only noticed `filter` param for alarms (cc @mlguerrero12)